history.txt

```diff
@@ -1,6 +1,93 @@
 OMERO version history
 =====================
 
+5.3.0-m5 (November 2016)
+------------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+-  database patches to add support for new administration roles
+-  removed restriction on name length
+-  added support for enumeration changes
+-  added new Grid View to both UI clients
+-  introduced the new Web API
+-  reactivated codomain support in the rendering Engine
+-  added support for Ice 3.6.3
+-  fixed graph issues when performing chown or moving between groups.
+
+5.3.0-m4 (September 2016)
+-------------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+-  completed Web decoupling
+-  improved CLI plugins
+-  deprecated Apache
+
+5.3.0-m3 (August 2016)
+----------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+-  added DB patches including support new options in rendering
+-  added support for LUT
+-  deprecated windows
+-  reorganized API documentation
+-  enabled plate download
+-  allowed customization of CSRF cookie
+-  fixed ROI related bug
+-  performed major code cleanup
+-  fixed cleanse
+
+5.3.0-m2 (April 2016)
+---------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.3.0. Use at your own risk**.
+
+Changes include:
+
+- added support for ROI folder at scale. Used IDR as a concrete example
```

Comment:
https://trello.com/c/LW8l7U8x/20-shape-attribute-renaming-cleanup is labelled for m2 too - worth adding something about this?